README.md

```diff
@@ -54,6 +55,15 @@ cordova build
 cordova run
 ```
 
+## Wayfinding
+
+Before using wayfinding part of the example, you need to create and download a graph from [IndoorAtlas web tool](). After you have added the graph to the example you can build and try it.
```

Comment:
... add link to documentation, e.g.:

    [create and download a graph](https://docs.indooratlas.com/manage/wayfinding/) from https://app.indooratlas.com

and an explicit comment about which file (`wayfinding-graph.json`) needs to be replaced with the downloaded graph